network-api/networkapi/wagtailpages/factory/publication.py

```diff
@@ -25,7 +18,7 @@
 
 
 class CollectionFactory(MP_NodeFactory):
-    name = Faker('text', max_nb_chars=60)
+    name = Faker("text", max_nb_chars=60)
```

Comment:
We should talk about this. Autopep might help here, but we "mostly" use single quotes rather than double quotes, so if we're doing a replacement pass, replacing things the other way would be better.